compiler/rustc_error_codes/src/error_codes/E0120.md

```diff
@@ -24,13 +32,13 @@ impl <T: MyTrait> Drop for MyWrapper<T> {
 
 ```
 
-Alternatively, wrapping trait objects requires something:
+Alternatively, `Drop` can be implemented on trait objects:
```

Comment:
Sorry, forgot to clarify it's not implementing the trait object directly. Should be something like this, feel free to reword

```suggestion
Alternatively, the `Drop` wrapper can contain a trait object:
```